presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kafka/kafka.properties

```diff
@@ -1,10 +1,10 @@
 connector.name=kafka
-kafka.table-names=product_tests.simple_key_and_value,\
```

Comment:
Techinically this commit does not split anything. Just rename.
So maybe commit message should be "Make naming regarding Kafka read tests more explicit"?